lib/backend/firestore/firestorebk_test.go

```diff
@@ -270,8 +270,13 @@ func TestDeleteDocuments(t *testing.T) {
 
 			lis, err := net.Listen("tcp", "localhost:0")
 			require.NoError(t, err)
-			go func() { require.NoError(t, srv.Serve(lis)) }()
-			t.Cleanup(srv.Stop)
+
+			errCh := make(chan error)
```

Comment:
```suggestion
			errCh := make(chan error, 1)
```

Should this one be buffered too?

Reply:
I think that would be safest! Good catch.